website/src/contribute/release-guide.md

```diff
@@ -167,8 +167,23 @@ please submit your GPG public key into [MIT PGP Public Key Server](http://pgp.mi
 
 #### Website development setup
 
-Prepare to update the Beam website by following the [website development
-instructions]({{ site.baseurl }}/contribute/website-contributions/).
+Updating the Beam website requires submitting PRs to both the main `apache/beam`
+repo and the `apache/beam-site` repo. The first contains most of the website
```

Comment:
Should be switched:

"The first contains reference manual generated from SDK code, while the second contains most of the website sources."

Reply:
done